test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -91,8 +95,7 @@ def test_safety(image):
         f"{image}", hide=True)
     try:
         run(f"{docker_exec_cmd} pip install safety yolk3k ", hide=True)
-        run_out = run(f"{docker_exec_cmd} safety check --json ", warn=True, hide=True)
-        json_str_safety_result = run_out.stdout
+        json_str_safety_result = safety_check.run_safety_check_on_container(docker_exec_cmd)
         safety_result = json.loads(json_str_safety_result)
         for package, affected_versions, curr_version, _, vulnerability_id in safety_result:
             run_out = run(f"{docker_exec_cmd} yolk -M {package} -f version ", warn=True, hide=True)
```

Comment:
Can this be run inside the SafetyCheck() object internally, if the object is given the `ignore_ids_list`?

Reply:
That might put too much control on the dlc package, and hence avoided that.